.generated.NoMobile.sln

```diff
@@ -180,6 +180,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "root", "root", "{233D34AB-9
 		CONTRIBUTING.md = CONTRIBUTING.md
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sentry.AspNetCore.Blazor", "src\Sentry.AspNetCore.Blazor\Sentry.AspNetCore.Blazor.csproj", "{7D797C46-EF23-40DD-A8AC-EB1DB3708EB0}"
```

Comment:
Is this specific to Wasm or this project will contain code for Server too?
If so we might need to name it `Sentry.AspNetCore.Blazor.Wasm` or `Sentry.AspNetCore.Blazor.WebAssembly` (whatever aligns with the upstream names, in .NET itself).

Reply:
Looks like `WebAssembly` is what .NET uses